### ISSUE TITLE
Allow changing number values with up/down keys.

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#1027]
## Feature Description

The editor should be able to change number values (such opacity, height, width, line-height etc.) via using keyboard up/down arrows.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by winterDroid** [comment]
Jon was testing to use the "numeric" input type which allows usage of the up/down keys but does not work with other symbols like %.

**Comment by swissspidy** [comment]
`numeric` input type does also not work with the `MULTIPLE_VALUE` constant used when multiple elements are selected.

We should just keep using regular input types, and implement arrow navigation ourselves using our keydown effects. Example:

```js
// form/numeric.js

const handleUpDown = useCallback(
  ({ key }) => {
    if (isMultiple) {
      return;
    }

    if (key === 'ArrowUp') {
      // Increment value
    } else if (key === 'ArrowDown') {
      // Decrement value
    }
  },
  [isMultiple]
);

// ...

useKeyDownEffect(
  ref,
  { key: ['up', 'down'] },
  handleUpDown,
  [handleUpDown]
);
```
